Makefile

```diff
@@ -141,6 +158,11 @@ KUSTOMIZE = $(shell pwd)/bin/kustomize
 kustomize: ## Download kustomize locally if necessary.
 	$(call go-get-tool,$(KUSTOMIZE),sigs.k8s.io/kustomize/kustomize/v3@v3.8.7)
 
+# TODO: Need to wait for skywalking-infra-e2e support to be installed by `go install` command
+# E2E = $(shell pwd)/bin/e2e
+# skywalking-e2e: ## Download skywalking-e2e locally if necessary.
+#	 $(call go-get-tool,$(E2E),github.com/apache/skywalking-infra-e2e@v1.1.0)
```

Comment:
FYI, now we can install via [`go install`](https://github.com/apache/skywalking-infra-e2e/blob/main/docs/en/setup/Install.md#install-via-go-install), however I don't want to patch a history version `v1.1.0` to make it possible to install via `go install` too. So you might want to use `go install github.com/apache/skywalking-infra-e2e/cmd/e2e@2a33478`, but if in any case you want to use a more "stable" version, I can start to release `v1.2.0`, just let me know.

Thanks for adopting skywalking-infra-e2e!!

Reply:
Great! I will test this method you suggested. Thx.

Reply:
I took your advice and it works fine 👍